### DISCUSSION TITLE
Honeycomb integration

### DISCUSSION THREAD

**Discussion by bluedread** [vercel/community#944] [Idea]
Hi folks, we are using Vercel and Honeycomb for a bit and we wonder if you have any tips/insights for how to handle monitoring/tracing for applications running in Vercel. 

We found this whitepaper and some tools that could help: 
* [Guide-Track_Core_Web_Vitals.pdf](https://www.honeycomb.io/wp-content/uploads/2022/07/Guide-Track_Core_Web_Vitals.pdf)
* [tracing-js](https://github.com/vercel/tracing-js)

These approaches seem to rely on including tools/libraries to the codebase of applications deployed to Vercel.

We'd rather like to enable something on a project- or account-level in Vercel instead of doing something on app/code-level. We wonder if there is something of a Honeycomb integration in the works that we could enable for projects? 

If any other folks have ideas about different approaches that would be appreciated as well.

**Comment by amyegan** [comment]
Thanks for reaching out @bluedread! You can find a directory of current integration options at [vercel.com/integrations](https://vercel.com/integrations). But I know the Integrations team is actively working to add more options to that list.

I can't say how quickly a UI integration for something like Honeycomb will become available. The most expedient solution at the moment is to make a code change

**Reply by bluedread** [reply]
Thanks @amyegan, we'll stick to code changes for now and keep an eye out for new integrations in the future. 👍
